lib/TelemetryFeed.jsm

```diff
@@ -257,10 +255,7 @@ this.TelemetryFeed = class TelemetryFeed {
    */
   get pingCentre() {
     Object.defineProperty(this, "pingCentre", {
-      value: new PingCentre({
-        topic: ACTIVITY_STREAM_ID,
-        overrideEndpointPref: ACTIVITY_STREAM_ENDPOINT_PREF,
```

Comment:
I noticed pingCentre is used in both `sendEvent` and `sendStructuredIngestionEvent` in the case of `sendEvent` it's being removed. In the case of `sendStructuredIngestionEvent` it's still using `pingCenter` So it's safe to remove `overrideEndpointPref` because it was only needed for the `sendEvent` usage, and it's however NOT needed in `sendStructuredIngestionEvent`'s usage of `pingCenter`?

Reply:
Correct, `overrideEndpointPref` is only used by PingCentre for Tiles. For the new data pipeline (`StructuredIngestion`), its endpoint will be specified by the function argument of `sendStructuredIngestionEvent`.